docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,628 @@ license: |
   limitations under the License.
 ---
 
-Aggregate functions
\ No newline at end of file
+Spark SQL provides build-in Aggregate functions defined in the dataset API and SQL interface. Aggregate functions
```

Comment:
nit: `Aggregate` -> `aggregate` along with the others? e.g.,  https://github.com/apache/spark/blame/master/docs/sql-ref-syntax-qry-select-having.md#L71

Reply:
thanks, changed.